toolchain/defs.bzl

```
@@ -230,11 +264,26 @@ zig_repository = repository_rule(
         "host_platform_sha256": attr.string_dict(),
         "url_formats": attr.string_list(allow_empty = False),
         "host_platform_ext": attr.string_dict(),
+        "exec": attr.string(),
     },
     environ = ["HERMETIC_CC_TOOLCHAIN_CACHE_PREFIX"],
     implementation = _zig_repository_impl,
 )
 
+def _local_zig_repository_impl(repository_ctx):
+    host_os = repository_ctx.os.name
+    host_arch = repository_ctx.os.arch
+    host_platform_compatible_zig = Label("@zig_sdk-{}-{}//:WORKSPACE".format(host_os, host_arch))
+    compatible_zig_path = repository_ctx.path(host_platform_compatible_zig)
+    repository_ctx.delete(".")
+    repository_ctx.symlink(str(compatible_zig_path) + "/..", ".")
+
+
+local_zig_repository = repository_rule(
```

```suggestion
host_zig_repository = repository_rule(
```
To avoid confusing with Bazel [local repository rules](https://bazel.build/rules/lib/repo/local).

---

Done